i18n/en.json

```diff
@@ -207,8 +207,11 @@
     "viewFolder": "View folder"
   },
   "SETTINGS": {
+    "applyAdditionalExtensions": "apply",
+    "additionalExtensions": "Support additional extensions",
     "autoGenerated": "Auto-generated tags",
     "buttonStyle": "Button style",
+    "cancelExtensionsEdit": "cancel",
```

Comment:
Thanks for alphabetizing too! 🙌 
I hope you're using a tool for that so you don't have to waste time 😉 
I like [sort lines](https://marketplace.visualstudio.com/items?itemName=Tyriar.sort-lines) and [sort JSON object](https://marketplace.visualstudio.com/items?itemName=richie5um2.vscode-sort-json) 🚀